src/app/modules/item/pages/item-details/item-details.component.ts

```diff
@@ -119,8 +119,12 @@ export class ItemDetailsComponent implements OnDestroy, BeforeUnloadComponent {
 
   private subscriptions = [
     this.itemDataSource.state$.subscribe(state => {
-      // reset task tabs when item changes.
+      // reset task tabs when item changes. By default do not display it unless we currently are on progress page
```

Comment:
item details should be "task" agnostic as it may be a chapter or a skill (>< task)

(apply this to everywhere there is "task tab" mentioned